CMakeLists.txt

```diff
@@ -374,6 +374,10 @@ elseif("${MD_SIM}" STREQUAL "LS1_MARDYN")
     target_link_libraries(mamico INTERFACE adios2::adios2)
   endif()
 endif()
+if(ENABLE_COVERAGE)
+  target_compile_options(mamico INTERFACE --coverage)
```

Comment:
Why did you remove `-O0` ? Isn't that needed for coverage?

Reply:
There is already -Og in Debug mode. Based on the documentation, it should be enough.

Reply:
Ok right